### ISSUE TITLE
make parseMetaphlanTSV() more general?

### ISSUE THREAD

**Issue by 2533245542** [lch14forever/microbiomeViz#5]
Could you edit a bit to make parseMetaphlanTSV() more general？ 

I am recently using .BIOM files for microbiome data analysis and they have similar taxonomy table format. The dataset can be easily visualized by GraPhlAn by simply pasting the dataset together with "." as a seperator, and use the generated txt file as input. 

If you want to see the dataset, do

biocLite("phyloseq")
library(phyloseq)
data("GlobalPatterns")
tax_table(GlobalPatterns)@.Data %>% View()

**Comment by 2533245542** [comment]
HI, I found that the update to phyloseq input support has been loaded. I try to run it but unfortunately my computer (MacBook Pro, Mac OS High Serria) does not successfully install the package, so I just copy the source code to run. Nevertheless, I still encounter an error: 
	Error: ggplot2 doesn't know how to deal with data of class list

Here are the files I used including the test file. In rgraphlan.R, I changed line 92 because of a previous error telling me that treedata() has an argument of “phy” instead of “phylo”. Could you take a look?

**Comment by 2533245542** [comment]
HI, this is the forward of the last email to your GitHub account, in case the last one did not deliver successfully.

**Comment by lch14forever** [comment]
Hi, I am currently in a conference. I will check it once I am back.

**Comment by lch14forever** [comment]
I have changed the phyloseq dependency to a lower version. Please try again and let me know if it works.

**Comment by 2533245542** [comment]
Yes, it is working now! There are some warnings when parsing phyloseq objects but I assume they are not that important.
